integration/testdata/TestNpmIntegration/run-tests.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
```

Comment:
Just wondering if this is unnecessarily restrictive. Won't this work on other shells?

Reply:
I don't think it is justifiable from an economical perspective to care about other shells, that's why I specify bash as a runtime.